stable/console-chart/templates/console-clusterrole.yaml

```diff
@@ -31,6 +31,7 @@ rules:
   - multiclusterhubs
   verbs:
   - get
+  - list
```

Comment:
whoops I think I may I have said `list`; but we can add `list` to, need to add `watch`